src/App.vue

```diff
@@ -19,6 +22,24 @@ import HelloI18n from './components/HelloI18n.vue';
   },
 })
 export default class App extends Vue {
+  client: AppGatewayServiceClient = undefined as any;
+
+  created() {
+    this.client = new AppGatewayServiceClient('http://localhost:50056', null, null);
```

Comment:
We should check it out if we can add an authentication header in `new AppGatewayServiceClient('http://localhost:50056', null, null);` instead of adding it as metadata per GRPC calls https://github.com/hwsc-org/hwsc-frontend/pull/21/files#r285904231